.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Thanks for contributing to go-client-mongodb-atlas!
+
+Please fill out the information below to help speed the review along, and hopefully
+the merge of your pull request!
+-->
+
+## Proposed changes
+
+<!-- 
+Describe the big picture of your changes here and communicate why we should accept this pull request. 
+If it fixes a bug or resolves a feature request, be sure to link to that issue. 
+-->
+
+<!--
+What go-client-mongodb-atlas issue does this PR address? (for example, #1234), remove this section if none
+-->
+
+Closes #[issue number]
+
+## Checklist
+
+<!--
+Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them,
+don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
```

Comment:
As people outside mongo starts to contribute is really important we make sure of this

Reply:
@gssbzn  - agree, we are getting more interest and when it makes sense we'll need this.  Do you happen to know where the agreement should get sent to?  I'll try to track down as well.  Once we know I'll approve and we can merge.

Reply:
@themantissa here's the details from the Kube project: 

"""
Contributing
Please file issues before filing PRs. For PRs to be accepted, contributors must sign our CLA.

Reviewers, please ensure that the CLA has been signed by referring to the contributors tool https://contributors.corp.mongodb.com/ (internal link).
"""

Reply:
^ we should probably put this into this project's repo, so just it's easy to find

Reply:
@themantissa @jamesbroadhead added to the CONTRIBUTING file tho the last section feels weird there

Reply:
I know - since it's an internal-only link. 
But I think without having it in the repo _somewhere_, our reviewers won't be able to find it easily. 
On balance, I prefer having it

Reply:
btw have the info from our legal team so will give this another read over to make sure it's in alignment and we'll be good!

Reply:
sgtm (either way - but I think the link needs to be _somewhere_ that's easily discoverable by reviewers)

Reply:
@themantissa moved there